dev/merge_spark_pr.py

```diff
@@ -92,9 +100,9 @@ def fail(msg):
 def run_cmd(cmd):
     print(cmd)
     if isinstance(cmd, list):
-        return subprocess.check_output(cmd)
+        return subprocess.check_output(cmd).decode(sys.stdout.encoding)
     else:
-        return subprocess.check_output(cmd.split(" "))
+        return subprocess.check_output(cmd.split(" ")).decode(sys.stdout.encoding)
```

Comment:
Is this changed?

Reply:
Yup, to use the system default encoding rather than utf-8 just in case some machines have other encodings. I roughly tested this too.

Reply:
Like, if a machine or OS has a different encoding, its standard output shouldn't be decoded into unicodes by utf-8. I think here is only the OS-specific code one.

Reply:
Got it~